src/main/scala-2.11/markets/orders/limit/LimitBidOrder.scala

```diff
@@ -19,7 +19,8 @@ import akka.actor.ActorRef
 
 import java.util.UUID
 
-import markets.orders.BidOrder
+import markets.orders.{AskOrder, BidOrder}
+import markets.orders.market.MarketAskOrder
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Imports should be sorted alphabetically](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1843449467/issues/source?bid=2921361&fileBranchId=3082852#l23)